go/dataprime-client.go

```diff
@@ -41,7 +41,11 @@ func (c DataprimeClient) Query(ctx context.Context, req *QueryRequest) (dataprim
 	defer conn.Close()
 	client := dataprime.NewDataprimeQueryServiceClient(conn)
 
-	return client.Query(callProperties.Ctx, req, callProperties.CallOptions...)
+	response, err := client.Query(ctx, req)
+	if err != nil {
+		return nil, NewSdkAPIError(err, dataprime.DataprimeQueryService_Query_FullMethodName, "dataprime")
```

Comment:
Please re-export the name as DataPrimeQueryRPC and use that. "dataprime" should be a constant too